### PR TITLE
Fix gh-413: Update supporting organizations

### LINKED ISSUE #413: Update About & Credits page
**About Page:**
https://scratch.mit.edu/about/

Update "Support and Funding" text to read: 

The Scratch project has received financial support from the National Science Foundation, Scratch Foundation, Google, LEGO Foundation, Intel, Cartoon Network, Lemann Foundation, and MacArthur Foundation. See the Credits <https://scratch.mit.edu/info/credits/> page for more information. If you'd like to support Scratch, please see the Scratch Foundation donate <https://secure.donationpay.org/scratchfoundation/> page, or contact us at donate@scratch.mit.edu

**Credits Page:**

Update Scratch Team section:
* Chris Garrity (chrisg on Scratch)
* Julia Zimmerman (jwzimmer1990 on Scratch)

Supporting Organizations should read:

National Science Foundation, Scratch Foundation, Google, LEGO Foundation, Intel, Cartoon Network, Lemann Foundation, MacArthur Foundation.

### PR DESCRIPTION
**Pull request by TheGrits**
Should resolve #413 

The only thing I wasn't too sure about was the url to Cartoon Network, the url I put in is the corporate url, not the actual Cartoon Network url.
@mewtaylor @kaschm @thisandagain @rschamp

### CODE REVIEW

**Comment by kaschm** [comment]
Hi @TheGrits! Can you switch the Cartoon Network URL to their parent company URL, http://www.turner.com/company ? Thanks!

**Comment by TheGrits** [comment]
@kaschm Done!

File: src/views/about/l10n.json [hunk 1]
```diff
@@ -22,5 +22,5 @@
...
     "about.support": "Support and Funding",
-    "about.supportDescription": "The Scratch project, initiated in 2003, has received generous support from the National Science Foundation (grants 0325828, 1002713, 1027848, 1019396), Intel Foundation, Microsoft, MacArthur Foundation, LEGO Foundation, Code-to-Learn Foundation, Google, Dell, Fastly, Inversoft, and MIT Media Lab research consortia. If you&#39;d like to support Scratch, please see our <a href=\"https://secure.donationpay.org/scratchfoundation/\">donate page</a>, or contact us at donate@scratch.mit.edu."
-}
\ No newline at end of file
+    "about.supportDescription": "The Scratch project has received financial support from the National Science Foundation, Scratch Foundation, Google, LEGO Foundation, Intel, Cartoon Network, Lemann Foundation, and MacArthur Foundation. See the <a href=\"/info/credits\">credits page </a> for more information. If you&#39;d like to support Scratch, please see the Scratch Foundation <a href=\"https://secure.donationpay.org/scratchfoundation/\">donate page</a>, or contact us at donate@scratch.mit.edu."
```
Comment (mewtaylor) [inline src/views/about/l10n.json]: @TheGrits looks like there might be an extra space in the `credits page` anchor.

Comment (TheGrits) [inline src/views/about/l10n.json]: Good catch, I just fixed it.

**Comment by mewtaylor** [comment]
@TheGrits I just added some more info on the Scratch Team members to add to the credits page too on https://github.com/LLK/scratch-www/issues/413. If you could add that into this PR, that'd be great!

**Comment by TheGrits** [comment]
@mewtaylor Done!

**Comment by mewtaylor** [comment]
@TheGrits looks great! just two small things:
1. On the `/about` page, [donate@scratch.mit.edu](mailto:donate@scratch.mit.edu) should be a mailto anchor
2. On the `/credits` page, I believe it's just "Intel" rather than "Intel Foundation" (is that correct, @kaschm?). Also, there appear to be missing spaces after the comma for Intel, Cartoon Network and Lemann.

Once those are finished up, this'll be ready to merge :).

**Comment by kaschm** [comment]
@mewtaylor @TheGrits - confirmed that it should read Intel and not Intel Foundation. Thanks!

**Comment by TheGrits** [comment]
@mewtaylor Done! 
@kaschm Thank you!

File: src/views/about/l10n.json [hunk 2]
```diff
@@ -22,5 +22,5 @@
...
     "about.support": "Support and Funding",
-    "about.supportDescription": "The Scratch project, initiated in 2003, has received generous support from the National Science Foundation (grants 0325828, 1002713, 1027848, 1019396), Intel Foundation, Microsoft, MacArthur Foundation, LEGO Foundation, Code-to-Learn Foundation, Google, Dell, Fastly, Inversoft, and MIT Media Lab research consortia. If you&#39;d like to support Scratch, please see our <a href=\"https://secure.donationpay.org/scratchfoundation/\">donate page</a>, or contact us at donate@scratch.mit.edu."
-}
\ No newline at end of file
+    "about.supportDescription": "The Scratch project has received financial support from the National Science Foundation, Scratch Foundation, Google, LEGO Foundation, Intel, Cartoon Network, Lemann Foundation, and MacArthur Foundation. See the <a href=\"/info/credits\">credits page</a> for more information. If you&#39;d like to support Scratch, please see the Scratch Foundation <a href=\"https://secure.donationpay.org/scratchfoundation/\">donate page</a>, or contact us at <a href=\"donate@scratch.mit.edu.\">donate@scratch.mit.edu</a>"
```
Comment (mewtaylor) [inline src/views/about/l10n.json]: @TheGrits the donate link should be a "mailto" link, sorry I didn't specify what I meant by that. It should look like:

`<a href=\"mailto:donate@scratch.mit.edu\">donate@scratch.mit.edu</a>.`

There also appears to be an extra period in the `href` that I think should come after the `</a>` tag.

**Comment by TheGrits** [comment]
@mewtaylor Should be fixed now, my bad. Sorry about that!

**Comment by mewtaylor** [comment]
@TheGrits oh please no apologies necessary :). Thanks the quick turnaround!

**Pull request merged**